### PR TITLE
Use the readme as dockerhub image description 

### PR DESCRIPTION
**Pull request by ad2ien**
Mainly fixes the dockerhub credential but also :
- improve the readme with badge logos, a docker and a contributing section
- Add a job to upload the readme on dockerhub as a image description

There should be a future PR about mentioning the config file in the readme.

### CODE REVIEW

**Review by amimart — APPROVED** [review]

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -72,3 +72,29 @@ jobs:
           push: true
           tags: ${{ steps.docker_metadata.outputs.tags }}
           labels: ${{ steps.docker_metadata.outputs.labels }}
+
+  dockerhub-description:
+    name: upload readme as dockerhub image description
```
Comment (ccamel) [inline .github/workflows/publish.yml:77]: Very minor suggestion (for consistency):

```suggestion
    name: Upload readme as dockerhub image description
```

**Review by bdeneux — APPROVED** [review]
Thanks ☺️

File: .github/workflows/publish.yml (hunk 1 above)
Comment (ccamel) [reply .github/workflows/publish.yml:77]: Sorry. After checking, the rule is to **not** specify a `name` for the `job`. Thus the jobs are named with their kebab key.

image

**Review by ingvaar — APPROVED** [review]

**Review by ccamel — APPROVED** [review]
thx!

**Pull request merged**